src/player-area.js

```diff
@@ -84,6 +85,24 @@ export const PlayerArea = () => {
       ]);
     });
 
+    socket.on("add token", async data => {
+      setTokens(tokens => tokens.filter(area => area.id !== data.id));
```

Comment:
These setTokens calls can be combined into one

Reply:
Changed in 2ab34f9.